skins/laika/src/components/pages/Contact.vue

```diff
@@ -0,0 +1,202 @@
+					<b-field>
+						<b-input type="text"
+								id="subject"
+								name="subject"
+								:placeholder="$t( 'contact_form_subject_placeholder' )"
+								v-model="formData.subject.value">
+						</b-input>
+					</b-field>
+					<span v-if="formData.subject.validity === Validity.INVALID" class="help is-danger">{{ $t( 'contact_form_subject_error' ) }}</span>
+				</div>
+				<div class="has-margin-top-18">
+					<label for="messageBody" class="subtitle">{{ $t( 'contact_form_body_label' ) }}</label>
+					<b-field>
+						<b-input type="textarea"
+								id="messageBody"
+								name="messageBody"
+								v-model="formData.comment.value">
+						</b-input>
+					</b-field>
+					<span v-if="formData.comment.validity === Validity.INVALID" class="help is-danger">{{ $t( 'contact_form_body_error' ) }}</span>
+				</div>
+			</fieldset>
+			<div class="has-margin-top-18">
+				<b-button id="submit-btn" type="is-primary is-main" native-type="submit">
+					{{ $t('contact_form_submit_button') }}
+				</b-button>
+			</div>
+		</form>
+	</div>
+</template>
+
+<script lang="ts">
+import Vue from 'vue';
+import { FormData } from '@/view_models/Contact';
+import { Helper } from '@/store/util';
+import { Validity } from '@/view_models/Validity';
+
+export default Vue.extend( {
+	name: 'Contact',
+	data: function (): { formData: FormData } {
+		return {
+			formData: {
+				name: {
+					name: 'name',
+					value: '',
+					pattern: '^.+$',
+					optionalField: true,
+					validity: Validity.VALID,
+				},
+				surname: {
+					name: 'surname',
+					value: '',
+					pattern: '^.+$',
+					optionalField: true,
+					validity: Validity.VALID,
+				},
+				donationNumber: {
+					name: 'donationNumber',
+					value: '',
+					pattern: '^[0-9]*$',
+					optionalField: true,
+					validity: Validity.VALID,
+				},
+				email: {
+					name: 'email',
+					value: '',
+					pattern: '^[^@]+@.+$',
+					optionalField: false,
+					validity: Validity.INCOMPLETE,
+				},
+				topic: {
+					name: 'topic',
+					value: null,
+					pattern: '^.+$',
+					optionalField: false,
+					validity: Validity.INCOMPLETE,
+				},
+				subject: {
+					name: 'subject',
+					value: '',
+					pattern: '^.+$',
+					optionalField: false,
+					validity: Validity.INCOMPLETE,
+				},
+				comment: {
+					name: 'comment',
+					value: '',
+					pattern: '(\n|.)+',
+					optionalField: false,
+					validity: Validity.INCOMPLETE,
+				},
+			},
+		};
+	},
+	props: [
+		'contactData',
+	],
+	methods: {
+		submit( event: Event ) {
+			let isValid = true;
+			Object.keys( this.$data.formData ).forEach( ( fieldName: string ) => {
+				let field = this.$data.formData[ fieldName ];
+				field.validity = Helper.inputIsValid( field.value, field.pattern, field.optionalField );
+				if ( field.validity !== Validity.VALID ) {
+					isValid = false;
+				}
+			} );
+			if ( isValid ) {
+				return true;
+			}
+			event.preventDefault();
```

Comment:
There are event modifiers in Vue which you can use directly in the template. 
In this case you can have `<form v-on:submit.prevent>` and remove this line from the ts.

Reply:
Cool :snowflake: